typescript/sdk/src/token/deploy.ts

```diff
@@ -124,6 +133,8 @@ abstract class TokenDeployer<
       isNativeTokenConfig(config)
     ) {
       return defaultArgs;
+    } else if (isCctpTokenConfig(config)) {
```

Comment:
This is not you and probably also not for this PR, but I wonder if we should have a different abstraction that encapsulates these instead of these if/else clauses